dist/threads-shared/lib/threads/shared.pm

```diff
@@ -8,7 +8,7 @@ use Config;
 
 use Scalar::Util qw(reftype refaddr blessed);
 
-our $VERSION = '1.68'; # Please update the pod, too.
+our $VERSION = '1.69'; # Please update the pod, too.
```

Comment:
The version number is reiterated in the POD on line 1.99, so probably (as per the comment) should be updated to 1.69.